### PR TITLE
update PR template

### PR DESCRIPTION
**Pull request by kevinhodges**
#### What are the links to relevant tickets?
N/A
#### What does this PR do?
Tweaks the grammar of the tempate
#### What unit tests does this PR have?
N/A
#### How should a developer review this?
Grammatical check should suffice
#### Any background context you want to provide?
The misspelling of 'succesfully' really annoys me...
#### Screenshots (if appropriate)
---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this cross browser for any visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.

#### Reviewer 1 @lukefrake
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running succesfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

#### Reviewer 2
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running succesfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

File: prTemplate.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #### What are the links to relevant tickets?
 #### What does this PR do?
-#### What unit tests does this PR have?
+#### What tests does this PR have?
```
Comment (lukefrake) [inline prTemplate.md]: How come?

Comment (kevinhodges) [inline prTemplate.md]: unit tests are not the only tests we write.

Comment (lukefrake) [inline prTemplate.md]: What other tests would we talk about here?

Comment (kevinhodges) [inline prTemplate.md]: The functional, end to end, regression, load or smoke tests?  Or anything else.  The point is that there must be tests.

Comment (lukefrake) [inline prTemplate.md]: But we wouldn't be talking about any of them at the moment as we aren't doing any of them?

**Comment by lukefrake** [comment]
:+1:

**Comment by sophiegeorge** [comment]
Spelling Bee 1st Place goes to Kevin.

**Pull request merged**